src/Clock.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+class Clock extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { date: new Date() };
+  }
+
+  componentDidMount() {
+    this.timerId = setInterval(() => {
+      this.setState({
+        date: new Date(),
+      });
+    }, 1000);
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.timerId);
+  }
+
+  render() {
+    return (
+      <p>
+        {this.state.date.toLocaleString("en-GB", {
```

Comment:
What if we make the language also dynamic and give the developer the option to input it in as needed?